tools/site_and_regional/default_data.cfg

```diff
@@ -0,0 +1,28 @@
+[main]
+clmforcingindir = /glade/p/cesmdata/inputdata
```

Comment:
Can this line be changed to use DIN_LOC_ROOT from cime?   This would make the file more machine independent.

Reply:
Actually, thinking about this it's not clear to me if this should be $DIN_LOC_ROOT or $CLM_USRDAT_DIR. But, either can function here.

Reply:
Erik will either work? Because python is reading this in as a config file I am not sure if those bash variables will be passed in. $CLM_USRDAT_DIR is intended to be the specific location of user-subsetted data. We want this to be the path to the non-subsetted data

Reply:
Ahhh, OK it should be $DIN_LOC_ROOT then. It doesn't need to be passed in as a bash variable -- it just needs to get the '$DIN_LOC_ROOT' string in the filenames in the user_nl_* files (and any files in shell_commmands).  That is for the user-mod-directory aspect of this. 

For anything that's opening files it will need to run `xmlquery --silent --value DIN_LOC_ROOT` and replace the $DIN_LOC_ROOT string with the output of that. So that's more work than I originally was thinking...

Reply:
I think I am confused Erik - this config file is currently telling the python script the location of the default files we plan on subsetting. This does not get used in any kind of case that one would create with this subset_data script.

In the future I do plan on using a dummy case to find the location of these files - which I image would make use of `DIN_LOC_ROOT`. If we want to run this script as it is now on any machine I think this would require a lot more work.

Perhaps I am not understanding you.

Reply:
Yes you are right. I was thinking about the user-mod directory side of this, but that's not what this is used for. @jedwards4b suggestion is really pointing out the desire for the future developments that you are talking about. So we should file his request under that. Sorry about that diversion, and adding a bunch of discussion here. My first impression was that it was a great idea (and it is just for a later development).

Reply:
OK, I'm going to resolve this conversation as it will be handled with #1597.

Reply:
Thanks @jedwards4b for this suggestion. I think it is a great suggestion; however, based on @adrifoster and @ekluzek it seems like simple using `$DIN_LOC_ROOT` would not work because this is a config file. 
I think with @adrifoster's suggestion and #1597 this would be resolved.